core/templates/pages/exploration-player-page/templates/lesson-information-card-modal.component.html

```diff
@@ -45,7 +45,6 @@ <h4 class="progress-bar-label-text">
           <button type="button"
                   class="btn btn-secondary save-progress-btn"
                   *ngIf="!userIsLoggedIn"
-                  [ngClass]="{'disabled oppia-disabled-link': checkpointStatusArray[0] === 'in-progress'}"
```

Comment:
Why this change?

Reply:
this was a part of another issue and its PR is open.. although I have removed this change from the current PR, it was pushed by mistake